ecosystem/indexer-grpc/indexer-grpc-utils/src/counters.rs

```diff
@@ -91,6 +93,8 @@ impl IndexerGrpcStep {
             // Table info service steps
             IndexerGrpcStep::TableInfoProcessedBatch => "1",
             IndexerGrpcStep::TableInfoProcessed => "2",
+            // DB tailer service steps
+            IndexerGrpcStep::DBTailerProcessed => "1",
```

Comment:
I guess the "internal indexer" should indeed be part of the general indexing framework? Indexer people do you think it's okay to mix them up? @grao1991  @bowenyang007  @jillxuu ?